### PR TITLE
Linked list

### PR DESCRIPTION
**Pull request by parjanyaacoder**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented Linked List in Java

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#400

### CODE REVIEW

**Review by siddharth25pandey — COMMENTED** [review]
@parjanyaacoder You cannot directly work on Master Branch, Create a develop branch to work on it.
I again request you to please go through the Contributions Guideline .
Thanks !

**Review by parjanyaacoder — COMMENTED** [review]
@siddharth25pandey I have created a branch called Linked List. I hope you noticed it.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@parjanyaacoder please remove unnecessary blank lines and whitespace. Also, try to follow [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html)

**Review by parjanyaacoder — COMMENTED** [review]
@iamrajiv Done

File: Java/ds/linkedList.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    
+    public void insert_At_beginning(int data) {
+    	Node ptr = new Node(data);
+    	if(head!=null)
+          {
+    		ptr.next=head;
+    		head=ptr;
+          }
+    	else 
+    	  {ptr.next=null;
```
Comment (vatsalkesarwani12) [inline Java/ds/linkedList.java]: Correct the indentation

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
void delete from end  function returning Null pointer exception when the linked list has 1 element

**Review by parjanyaacoder — COMMENTED** [review]
I think it will work now.

File: Java/ds/linkedList.java (hunk 1 above)
Comment (parjanyaacoder) [reply Java/ds/linkedList.java]: > Correct the indentation
Pls review now.

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
@parjanyaacoder delete at position function giving null pointer exception when only 1 element present in the list

**Comment by parjanyaacoder** [comment]
Please review.
Thanks to @iamrajiv @siddharth25pandey @plazzy99 for helping me in correcting this code.

**Review by siddharth25pandey — APPROVED** [review]
@parjanyaacoder looks fine to me
Able to merge.
Thanks !

**Review by vatsalkesarwani12 — APPROVED** [review]
Looks fine 
All the mentioned issues resolved.

**Review by iamrajiv — APPROVED** [review]

**Comment by parjanyaacoder** [comment]
@iamrajiv Please merge this pr.

**Pull request merged**